Cargo.toml

```
@@ -26,6 +26,7 @@ members = ["examples"]
 [dependencies]
 num = { version = "0.3.0", default-features = false }
 serde = { version = "1.0.0", default-features = false, features = ["derive"], optional = true }
+defmt = { version = "0.2", optional = true }
```

It is a special case, but I would enforce `v0.2.3` and higher, because ithis could happen:  https://github.com/stm32-rs/bxcan/pull/45 (though that case is unlikely to happen outside if explicitly testing for it)
```suggestion
defmt = { version = "0.2.3", optional = true }
```